bin/i18n/resources/pegasus/emails/sync_out.rb

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env ruby
+
+require 'fileutils'
+
+require_relative '../../../i18n_script_utils'
+require_relative '../../../utils/pegasus_markdown'
+require_relative '../emails'
+
+module I18n
+  module Resources
+    module Pegasus
+      module Emails
+        class SyncOut < I18n::Utils::SyncOutBase
+          DIR_NAME = 'emails'.freeze
+
+          def execute
+            progress_bar.start
+
+            I18nScriptUtils.process_in_threads(pegasus_languages) do |pegasus_lang|
+              crowdin_locale_dir = I18nScriptUtils.locale_dir(pegasus_lang[:crowdin_name_s])
+              crowdin_locale_resource_dir = File.join(crowdin_locale_dir, DIR_NAME)
+              next unless File.directory?(crowdin_locale_resource_dir)
+
+              locale = pegasus_lang[:locale_s]
+              distribute(locale, crowdin_locale_resource_dir) unless locale == 'en-US'
+
+              FileUtils.rm_r(crowdin_locale_resource_dir)
+            ensure
+              I18nScriptUtils.remove_empty_dir(crowdin_locale_dir)
+
+              mutex.synchronize {progress_bar.increment}
+            end
+
+            progress_bar.finish
+          end
+
+          def process(language)
+            crowdin_file_path = I18nScriptUtils.locale_dir(language[:crowdin_name_s], DIR_NAME, FILE_NAME)
+            return unless File.file?(crowdin_file_path)
+
+            unless I18nScriptUtils.source_lang?(language)
+              # Distributes the localization
+              target_i18n_file_path = File.join(ORIGIN_I18N_DIR_PATH, "#{language[:locale_s]}.yml")
```

Comment:
Watch out, I know you were looking at Dashboard examples, so you will have to update the name and format of the files synced out. Something along these lines:
```suggestion
              target_i18n_file_name = crowdin_file_name.gsub(/en.md/,"#{language[:locale_s]}.md")
              target_i18n_file_path = File.join(ORIGIN_I18N_DIR_PATH, target_i18n_file_name)
```

Reply:
Oh gotcha, thank you!